docs/whats-new/release-notes/v2_15_0.md

```diff
@@ -0,0 +1,139 @@
+
+#### Zowe Explorer API
+
+- See the [Zowe Explorer API](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer-api/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer FTP Extension
+
+- See the [Zowe Explorer FTP Extension](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer-ftp-extension/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer ESLint Plug-in
+
+- See the [Zowe Explorer ESLint Plug-in](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/eslint-plugin-zowe-explorer/CHANGELOG.md) changelog for updates included in this release.
+
+## Bug fixes
+
+Zowe Version 2.15.0 contains the bug fixes that are described in the following topics.
+
+### Zowe installation and packaging
+ 
+### Zowe Application Framework
+
+#### Zlux Server Framework
+
+### Zowe API Mediation Layer
+
+ -Displaying the base path in API Catalog has been fixed. ([#3297](https://github.com/zowe/api-layer/issues/3297))
+- Disabling EhCache using `apiml.caching.enabled=false` has been fixed. ([#3280](https://github.com/zowe/api-layer/issues/3280))
+
+### Zowe CLI
+
+#### Zowe CLI (Core)
+
+- Fixed race condition in the `config convert-profiles` command that may fail to delete secure values for old profiles.
+
+- Resolved technical currency by updating `socks` transitive dependency. ([#2049](https://github.com/zowe/zowe-cli/pull/2049))
+
+- Updated zos-files copy dsclp system tests to include large mock files. This improves system test coverage for edge cases involving large data sets. ([#2023](https://github.com/zowe/zowe-cli/pull/2023))
+
+#### Zowe CLI Imperative Framework
+
+- Fixed race condition in the `config convert-profiles` command that may fail to delete secure values for old profiles. ([Zowe CLI #2055](https://github.com/zowe/zowe-cli/pull/2055))
+
+- Resolved issue in `ProfileInfo` where schema comparisons fail, specifically when comparing the cached schema against a command-based schema during registration. ([Zowe CLI 2045](https://github.com/zowe/zowe-cli/pull/2045))
+
+- Resolved technical currency by updating `socks` transitive dependency. ([Zowe CLI 2049](https://github.com/zowe/zowe-cli/pull/2049))
+
+- Updated `mustache` and `jsonschema` dependencies for technical currency. ([Zowe CLI 2024](https://github.com/zowe/zowe-cli/pull/2024))
+
+- Fixed issue where secure property names could be returned for the wrong profile. ([Zowe Explorer #2633](https://github.com/zowe/vscode-extension-for-zowe/issues/2633))
+
+- Fixed issue when a property is not found in `ProfileInfo.updateProperty({forceUpdate: true})` so that extenders can set or update a property in a team configuration even if it is not defined in the schema without their application/extension failing. ([Zowe Explorer #2493](https://github.com/zowe/vscode-extension-for-zowe/issues/2493))
+
+#### IMS Plug-in for Zowe CLI
+
+- Added missing shrinkwrap.
```

Comment:
@adam-wolfe -- sorry, one more: what would be the PR/issue link for this? and what does this magical shrinkwrap do in this instance?

Reply:
https://github.com/zowe/zowe-cli-ims-plugin/pull/65 - the npm shrinkwrap is used to specify specific versions of dependencies that are installed with the plug-in.